### ISSUE TITLE
Error message show only file name instead of full path

### ISSUE THREAD

**Issue by wsporto** [denoland/deno#5202]
When I import any deno module (for ex, the parse module), the error message show only the file name instead of the full path of the source file. That avoid the vscode from creating a link where I can click and jump to the line of the error.

![image](https://user-images.githubusercontent.com/26843533/81490998-0e702380-925f-11ea-8528-ff8f556ea56f.png)

Without importing anything, the full path of the file is shown as expected.

![image](https://user-images.githubusercontent.com/26843533/81491011-33fd2d00-925f-11ea-9966-33e6a082222a.png)

**Comment by waldyrious** [comment]
A relative link would also work, and would be more readable IMO:

```console
$ deno run foo.ts
error: Uncaught Error: A custom error msg
throw new Error("A custom error msg");
      ^
    at ./foo.ts:3:7
```

**Comment by apiel** [comment]
I don't know if it is related but since today the fullpath is missing in the stacktrace in my github [CI](https://github.com/apiel/adka/runs/814693690?check_suite_focus=true):

![image](https://user-images.githubusercontent.com/5873198/85932819-dafd5d00-b8cf-11ea-86bf-88614babb38f.png)

**Comment by nayeemrmn** [comment]
Fixed by #9462.
